pkgs/games/vapor/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, fetchurl, unzip, love, lua, makeWrapper, makeDesktopItem }:
+
+let
+  pname = "vapor";
+  version = "0.2.3";
+  commitid = "dbf509f";
+
+  icon = fetchurl {
+    url = "http://vapor.love2d.org/sites/default/files/vapT240x90.png";
+    sha256 = "1xlra74lpm1y54z6zm6is0gldkswp3wdw09m6a306ch0xjf3f87f";
+  };
+
+  desktopItem = makeDesktopItem {
+    name = "Vapor";
+    exec = "${pname}";
+    icon = "${icon}";
+    comment = "LÖVE Distribution Client"; 
+    desktopName = "Vapor";
+    genericName = "vapor";
+    categories = "Game;";
+  };
+
+in 
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url =
+    "https://github.com/josefnpat/${pname}/releases/download/${version}/${pname}_${commitid}.love";
+    sha256 = "0w2qkrrkzfy4h4jld18apypmbi8a8r89y2l11axlv808i2rg68fk";
+  };
+
+  nativeBuildInputs = [ lua love ];
+  buildInputs = [ makeWrapper ];
```

Comment:
These are switched. `nativeBuildInputs` should be `buildInputs` and the other way around.